lib/active_merchant/billing/gateways/fat_zebra.rb

```diff
@@ -44,6 +44,36 @@ def purchase(money, creditcard, options = {})
         commit(:post, 'purchases', post)
       end
 
+      # To create an authorization on a credit card use:
+      #
+      #   authorize(money, creditcard)
+      #
+      # To authorize a tokenized card
+      #
+      #   authorize(money, "abzy87u", :cvv => "123")
+      def authorize(money, creditcard, options = {})
+        post = {}
+
+        add_amount(post, money, options)
+        add_creditcard(post, creditcard, options)
+        post[:capture] = false
+        post[:reference] = options[:order_id]
+        post[:customer_ip] = options[:ip]
+
+        commit(:post, 'purchases', post)
+      end
+
+      # Capture a previous authorization
+      #
+      # money - Integer - the amount to capture
+      # txn_id - String - the original transaction to be captured
+      def capture(money, txn_id, options = {})
```

Comment:
Minor nitpick, but can you use `authorization` instead of `txn_id` for the param name? And there's no reason to comment on what these methods do, they're part of the core ActiveMerchant API and should be well understood by anyone using them.